debian/control.in

```diff
@@ -0,0 +1,27 @@
+Source: pgautofailover
+Section: database
+Priority: optional
+Maintainer: Citus Data <packaging@citusdata.com>
+Uploaders: Nils Dijk <nils@citusdata.com>
+Build-Depends: debhelper (>= 9), autotools-dev, flex, postgresql-server-dev-all (>= 158~), libedit-dev, libpam0g-dev, libselinux1-dev, libxslt1-dev
+Standards-Version: 3.9.7
+Homepage: https://github.com/citusdata/pg_auto_failover
+
+Package: pgautofailover-cli
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends}
+Provides: pgautofailover-cli
+Conflicts: pgautofailover-cli
+Recommends: postgresql-10-pgautofailover | postgresql-11-pgautofailover
+Description: pg_auto_failover CLI
+ This CLI is used to manage pg_auto_failover installations to provide High
+ Availability to Postgres.
+
+Package: postgresql-PGVERSION-pgautofailover
```

Comment:
package name for the extension, this will be what the user installs as it depends on the cli, that will be installed as well.

Reply:
Similarly, can we do either `postgresql-PGVERSION-pg-auto-failover` or `postgresql-PGVERSION-auto-failover` (since the pg is already in the package name for debian)